ocrd/ocrd/workspace.py

```diff
@@ -387,21 +389,23 @@ def image_from_segment(self, segment, parent_image, parent_xywh):
             # or reduced polygon coordinates).
             if 'angle' in segment_xywh and segment_xywh['angle']:
                 log.info("About to rotate segment '%s' by %.2f°",
-                          segment.id, segment_xywh['angle'])
+                         segment.id, segment_xywh['angle'])
                 segment_image = segment_image.rotate(segment_xywh['angle'],
                                                      expand=True,
-                                                     #resample=Image.BILINEAR,
+                                                     # resample=Image.BILINEAR,
                                                      fillcolor='white')
         # subtract offset from any increase in binary region size over source:
-        segment_xywh['x'] -= round(0.5 * max(0, segment_image.width  - segment_xywh['w']))
-        segment_xywh['y'] -= round(0.5 * max(0, segment_image.height - segment_xywh['h']))
+        segment_xywh['x'] -= round(0.5 * max(0,
+                                             segment_image.width - segment_xywh['w']))
+        segment_xywh['y'] -= round(0.5 * max(0,
+                                             segment_image.height - segment_xywh['h']))
         return segment_image, segment_xywh
 
     # pylint: disable=redefined-builtin
     def save_image_file(self, image,
                         file_id,
                         page_id=None,
-                        file_grp='OCR-D-IMG', # or -BIN?
+                        file_grp='OCR-D-IMG',  # or -BIN?
```

Comment:
This interesting since the bags already contain images in the directory `OCR-D-IMG`.

Reply:
Yes, by convention they always do. So maybe it would be better to use `None` as default here, to force users of this function to think about _useful_ file groups (hopefully set via their output fileGrp)?

Reply:
> So maybe it would be better to use None as default here, to force users of this function to think about useful file groups 

Change the signature to 

```python
 def save_image_file(self, image,
                        file_id,
                        file_grp,
                        page_id=None,
                        format='PNG',
                        force=True)
```
?

Reply:
Yes, at this moment we are most likely the only users of that function, so let's change the signature like that!

Reply:
Done in https://github.com/OCR-D/core/pull/303/commits/fc1c0064aee35d5afe2e25d7c90401d563be84fe